### PR TITLE
TASK: Remove deprecated beta10 apis

### LINKED ISSUE #5043: Remove `$node->subgraphIdentity->contentStreamId`
Subtask of https://github.com/neos/neos-development-collection/issues/4564

The `contentStreamId` is currently still used

- [x] in Neos.Seo
  -> https://github.com/neos/neos-seo/pull/187
- [x] in the Node
- [x] in the NodeAggregate
   - [x] all usages removed
- [x] as cache identifier for subgraphs (in its identity)
- [x] in property adjustments for publishing events based from a node
- [x] in the e2e test in the `NodeDiscriminator::fromNode`
  -> https://github.com/neos/neos-development-collection/pull/5169
- [x] in the `NodeCacheEntryIdentifier` and `Neos.Caching.entryIdentifierForNode`
  -> https://github.com/neos/neos-development-collection/pull/5083 
- [x] in the old legacy `NodeAddressFactory`
  -> https://github.com/neos/neos-development-collection/pull/5068
- [x] in the Fusion `CacheTag` -> https://github.com/neos/neos-development-collection/issues/4810
- [x] for equals check against `AssetUsageReference` (because the asset usage stuff is not workspace aware yet)
  - [x] will be fixed via https://github.com/neos/neos-development-collection/issues/5084 , but as the used part is currently broken either way we can go ahead: https://github.com/neos/neos-development-collection/issues/5145

### PR DESCRIPTION
**Pull request by mhsdesign**
Resolves: https://github.com/neos/neos-development-collection/issues/5043

- Remove b/c layer `CommandResult::block()`
- Remove b/c layer `Node::$nodeName`
- Remove b/c layer `Node::$nodeAggregateId`
- Remove b/c layer `Node::getLabel()`
- Remove b/c layer `Node::$subgraphIdentity` 
- Remove `NodeAggregate::$contentStreamId`
- The b/c layer `Node::$nodeType` was already removed previously via https://github.com/neos/neos-development-collection/pull/5021

**Upgrade instructions**

See release notes of Beta10: https://discuss.neos.io/t/neos-9-beta-10-release/6578

**Review instructions**

The last usage of `$node->subgraphIdentity->contentStreamId` in the CI will be removed here https://github.com/neos/neos-development-collection/pull/5169 and will turn the tests green.

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [ ] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

### CODE REVIEW

**Comment by mhsdesign** [comment]
The CI will be made green via https://github.com/neos/neos-development-collection/pull/5169 (could probably be merged first)

File: Neos.Neos/Resources/Private/Fusion/Backend/Views/Login.fusion [hunk 1]
```diff
@@ -46,7 +46,7 @@ prototype(Neos.Neos:View.Login) < prototype(Neos.Fusion:Component) {
               </figure>
 
               <h1 class="neos-login-heading">
-                {I18n.id('login.index.title').value('Login to').package('Neos.Neos').source('Main')} <strong>{site.name}</strong>
+                {I18n.id('login.index.title').value('Login to').package('Neos.Neos').source('Main')} <strong>{props.site.name.value}</strong>
```
Comment (grebaldi) [inline Neos.Neos/Resources/Private/Fusion/Backend/Views/Login.fusion:49]: Nice catch!

File: Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php [hunk 2]
```diff
@@ -127,29 +93,19 @@ private function __construct(
...
-            $dimensionSpacePoint,
-            $visibilityConstraints
-        );
     }
 
     /**
      * @internal The signature of this method can change in the future!
      */
-    public static function create(ContentRepositoryId $contentRepositoryId, WorkspaceName $workspaceName, DimensionSpacePoint $dimensionSpacePoint, NodeAggregateId $aggregateId, OriginDimensionSpacePoint $originDimensionSpacePoint, NodeAggregateClassification $classification, NodeTypeName $nodeTypeName, PropertyCollection $properties, ?NodeName $nodeName, NodeTags $tags, Timestamps $timestamps, VisibilityConstraints $visibilityConstraints, ContentStreamId $contentStreamId): self
+    public static function create(ContentRepositoryId $contentRepositoryId, WorkspaceName $workspaceName, DimensionSpacePoint $dimensionSpacePoint, NodeAggregateId $aggregateId, OriginDimensionSpacePoint $originDimensionSpacePoint, NodeAggregateClassification $classification, NodeTypeName $nodeTypeName, PropertyCollection $properties, ?NodeName $nodeName, NodeTags $tags, Timestamps $timestamps, VisibilityConstraints $visibilityConstraints): self
```
Comment (grebaldi) [inline Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php]:
```suggestion
    public static function create(ContentRepositoryId $contentRepositoryId, WorkspaceName $workspaceName, DimensionSpacePoint $dimensionSpacePoint, NodeAggregateId $aggregateId, OriginDimensionSpacePoint $originDimensionSpacePoint, NodeAggregateClassification $classification, NodeTypeName $nodeTypeName, PropertyCollection $properties, ?NodeName $name, NodeTags $tags, Timestamps $timestamps, VisibilityConstraints $visibilityConstraints): self
```

for consistency (while we're at it)

File: Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php [hunk 3]
```diff
@@ -127,29 +93,19 @@ private function __construct(
...
     {
-        return new self($contentRepositoryId, $workspaceName, $dimensionSpacePoint, $aggregateId, $originDimensionSpacePoint, $classification, $nodeTypeName, $properties, $nodeName, $tags, $timestamps, $visibilityConstraints, $contentStreamId);
+        return new self($contentRepositoryId, $workspaceName, $dimensionSpacePoint, $aggregateId, $originDimensionSpacePoint, $classification, $nodeTypeName, $properties, $nodeName, $tags, $timestamps, $visibilityConstraints);
```
Comment (grebaldi) [inline Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php]:
```suggestion
        return new self($contentRepositoryId, $workspaceName, $dimensionSpacePoint, $aggregateId, $originDimensionSpacePoint, $classification, $nodeTypeName, $properties, $name, $tags, $timestamps, $visibilityConstraints);
```

**Review by grebaldi — APPROVED** [review]
I've got nothin' but nitpicks :) And aside from the remaining linting issue, that's a :+1: from me.

I also did an E2E UI test run locally and it looks :ok_hand:

File: Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php (hunk 2 above)
Comment (mhsdesign) [reply Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php]: adjusted :)

**Review by nezaniel — APPROVED** [review]
Tests are green again (thx @kitsunet), this looks very mergeable to me

**Comment by kitsunet** [comment]
Nod yeah post merge 👍

**Pull request merged**